scripts/qa_crowbarsetup.sh

```diff
@@ -3902,6 +3924,33 @@ function oncontroller_testsetup()
     nova stop "$instanceid"
     wait_for 100 1 "test \"x\$(nova show \"$instanceid\" | perl -ne 'm/ status [ |]*([a-zA-Z]+)/ && print \$1')\" == xSHUTOFF" "testvm to stop"
 
+    # run tests for Magnum bay deployment
+    if [ -n "$want_magnum" ]; then
+        if ! magnum baymodel-show susek8sbaymodel; then
+            safely magnum baymodel-create --name susek8sbaymodel \
+                --image-id SLE12SP1-JeOS-k8s-magnum \
+                --keypair-id controller-ssh \
```

Comment:
I think you should use the ```default``` keypair. that's the one we also use for the tempest tests